### DISCUSSION TITLE
Wyszukiwanie użytkowników powinno być na backendzie i zwracane frontendowi

### DISCUSSION THREAD

**Discussion by itchystone** [emkarcinos/WMIAdventure#560] [Codding]
Wyszukiwanie użytkowników po nazwie będzie musiało być jakoś na backendzie zrobione, żeby to wyszukiwanie działało razem z paginacją

Narazie backend zwraca jakąś część użytkowników i frontend tylko z tej części dopasowuje nazwę

Powinno być tak, że frontend wysyła query do backendu "daj mi wszystkich userów co mają w nazwie apple" i z tego powinna być paginacja

**Comment by emkarcinos** [comment]
Zdecydowanie się nie zgadzam. Ilość requestów wysyłana do backendu przy wpisywaniu / usuwaniu kolejnego znaku w textboxie jest za duża, jakby kilku userów na raz chciało wyszukać coś w textboxie to by się serwer zawiesił XD

**Reply by mattyl006** [reply]
zgadzam się

**Comment by emkarcinos** [comment]
Ja generalnie mam pomysł jak zrobić to wyszukiwanie na frontendzie i mogę to zrobić tak jak trzeba

**Reply by mattyl006** [reply]
spoko, ale to nie priorytet, zróbmy to na opt'a
